response_operations_ui/controllers/case_controller.py

```diff
@@ -94,10 +94,11 @@ def get_case_groups_by_business_party_id(business_party_id):
     return response.json()
 
 
-def get_cases_by_business_party_id(business_party_id):
+def get_cases_by_business_party_id(business_party_id, max_case_per_survey):
     logger.info('Retrieving cases', business_party_id=business_party_id)
     url = f'{app.config["CASE_URL"]}/cases/partyid/{business_party_id}'
-    response = requests.get(url, auth=app.config['CASE_AUTH'], params={"iac": "True"})
+    response = requests.get(url, auth=app.config['CASE_AUTH'],
+                            params={"iac": "True", "max_cases_per_survey": max_case_per_survey})
```

Comment:
It's odd that we get `app.config['CASE_AUTH']` here but pass in `app.config['MAX_CASES_RETRIEVED_PER_SURVEY']` as a parameter. Would like to see this consistent in one direction or the other

Reply:
Disagree on this one . This is consistent across the app. Auth  , headers and params are set distinctly

Reply:
I would like to see this changed, tagging @necrophonic for resolution